analysis/report_templates/default.html

```diff
@@ -142,8 +142,10 @@ <h5 class="center-align">Mann-Whitney U test</h4>
         </div> <!-- id="{{ benchmark.name }}" -->
         {% endfor %}
 
-        <div class="divider"></div>
-        See the <a href="data.csv.gz">raw data</a> for this report.
+        <div id="data" class="section scrollspy">
+            <h2 class="green-text text-darken-3">experiment data</h2>
```

Comment:
consider: "raw experiment data in csv"

Reply:
feels simple is better, also matches style "experiment summary"